package.json

```diff
@@ -3,5 +3,13 @@
   "license": "MIT",
   "engines": {
     "node": "5.10"
+  },
+  "scripts": {
+    "postinstall": "cd client && npm install",
+    "clean": "rm -f app/assets/webpack/*"
+  },
+  "cacheDirectories": ["client/node_modules"],
```

Comment:
Is this an override or addition? If it's an override then do we intentionally skip the current node_modules?